src/fragments/lib/auth/js/enable-signin-signup-signout.mdx

```diff
@@ -1,8 +1,12 @@
-In this guide you will set up sign-up, sign-in, and sign-out and then test this functionality. This includes a review of sign-up methods provided by Amplify, the specific user attributes used by Amazon Cognito, how to confirm users after initial sign-up, and the differences between local and global sign-out.
+> *This guide provides essential information to help you select and complete the activities you need for your Amplify project. We organized each section around a distinct job or decision point to help you understand your available options, steps to complete, and recommended best practices.*
```

Comment:
This looks like I changed more than I did - I added the JTBD Guide info at the top which cascaded changes downward.

Reply:
cool!